doc/index.md

```diff
@@ -1,56 +1,59 @@
+Npgsql has an Entity Framework (EF) Core provider. It behaves like other EF Core provider (e.g. SQL Server), so all of the information in the [general EF Core docs](https://docs.microsoft.com/en-us/ef/core/index) applies. If you're just getting started with EF Core, those docs are the best place to start.
 
 Development happens in the [Npgsql.EntityFrameworkCore.PostgreSQL](https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL) repository, all issues should be reported there.
 
-## Using the Npgsql EF Core Provider
+## Configuring the project file
 
-To use the Npgsql EF Core provider, simply add a dependency on `Npgsql.EntityFrameworkCore.PostgreSQL`. You can follow the instructions in the general [EF Core Getting Started docs](https://docs.microsoft.com/en-us/ef/core/get-started/).
+To use the Npgsql EF Core provider, add a dependencies for `Npgsql` and `Npgsql.EntityFrameworkCore.PostgreSQL`. You can follow the instructions in the general [EF Core Getting Started docs](https://docs.microsoft.com/en-us/ef/core/get-started/).
 
-Following is an example (new-style) csproj using Npgsql EF Core:
+Below is a `.csproj` file for a console application that uses the Npgsql EF Core provider:
 
 ```xml
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.0.0" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Design" Version="2.0.0" />
-  </ItemGroup>
-  <ItemGroup>
-    <DotNetCliToolReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.0" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore" Version="2.1.0" />
+    <PackageReference Include="Npgsql" Version="4.0.0" />
+    <PackageReference Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="2.1.0" />
   </ItemGroup>
 </Project>
 ```
 
-## Using the Npgsql provider
+## Defining a `DbContext`
 
-    public class BlogContext : DbContext
-    {
-        // When used with ASP.net core, add these lines to Startup.cs
-        //   var connectionString = Configuration.GetConnectionString("BlogContext");
-        //   services.AddEntityFrameworkNpgsql().AddDbContext<BlogContext>(options => options.UseNpgsql(connectionString));
-        // and add this to appSettings.json
-        // "ConnectionStrings": { "BlogContext": "Server=localhost;Database=blog" }
+```c#
+public class BlogContext : DbContext
+{
+    public BlogContext(DbContextOptions<BlogContext> options) : base(options) { }
         
-        public BlogContext(DbContextOptions<BlogContext> options) : base(options) { }
-        
-        public DbSet<BlogPost> BlogPosts { get; set; }     
-    }
-    
+    public DbSet<BlogPost> BlogPosts { get; set; }     
```

Comment:
Let's include the `Blog` and `Post` classes exactly as in [the EF Core getting started docs](https://docs.microsoft.com/en-us/ef/core/get-started/netcore/new-db-sqlite), for clarity.